libfs/fs.go

```diff
@@ -30,6 +31,7 @@ type FS struct {
 	root     libkbfs.Node
 	h        *libkbfs.TlfHandle
 	subdir   string
+	uniqID   string
```

Comment:
No, this is a unique ID for only this instance of the remote-helper.  There could be multiple GRH processes on the same device, and they all need to have different uniqIDs.  It's used for making temporary files in KBFS that need to have globally unique names.

LockIDs need to be agreed on by all devices and all processes, otherwise they wouldn't be very effective locks!